src/widget/wtracktableview.cpp

```diff
@@ -814,7 +816,11 @@ void WTrackTableView::hideOrRemoveSelectedTracks() {
             return;
         }
 
-        response = QMessageBox::question(this, tr("Confirm track removal"), message);
+        response = QMessageBox::question(this,
```

Comment:
how would that part be reached given the return; in the else branch above?

Reply:
The `return;` above is for the case when the track model does not support hiding nor removing tracks (e.g. in Missing tracks or Hidden tracks).

Reply:
uhaa, I lost track in the condesnsed mobile view. foregt evrything I said :\